File: Java/백준/Bronze/2953. 나는 요리사다/나는 요리사다.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int maxsum = 0;
        int num = 0;
        for (int i=0;i<5;i++){
            int sum =0;
            for (int j=0;j<4;j++){
                sum += in.nextInt();
            }
            if (sum>maxsum){
                num = i+1;
                maxsum = sum;
            }
        }
        System.out.printf("%d %d",num,maxsum);
    }
}

File: C++11/백준/Bronze/2953. 나는 요리사다/나는 요리사다.cc
#include <iostream>
using namespace std;

int main(){
    int a,b,c,d,maxsum=0,num=0;
    for (int i=0;i<5;i++){
        cin >> a>>b>>c>>d;
        int sum = a+b+c+d;
        if(sum>maxsum){
            maxsum=sum;
            num=i+1;
        }
    }
    cout << num << " " << maxsum;
}